internal/provider/interactive-install.go

```diff
@@ -15,15 +15,38 @@ func InteractiveInstall(e *pluggable.Event) pluggable.EventResponse { //nolint:r
 			YAMLSection: "p2p.network_token",
 			Prompt:      "Insert a network token, leave empty to autogenerate",
 			AskFirst:    true,
-			AskPrompt:   "Do you want to setup a full mesh-support?",
+			AskPrompt:   "Do you want to set up a full mesh-support?",
 			IfEmpty:     node.GenerateNewConnectionData().Base64(),
 		},
 		{
 			YAMLSection: "k3s.enabled",
 			Bool:        true,
-			Prompt:      "Do you want to enable k3s?",
+			Prompt:      "Do you want to enable k3s? (Select either k3s or k0s)",
```

Comment:
@kairos-io/maintainers I wonder how we should go about the interactive installer. Ideally, it should be more like, do you want K8s installed, select a distro: k3s, k0s or something like that.

Reply:
Up to now, different k8s distributions were handled by different providers altogether. E.g. [provider-kubeadm](https://github.com/kairos-io/provider-kubeadm/), [provider-rke2](https://github.com/kairos-io/provider-rke2), etc.

Why should we handle k0s differently?

Reply:
@jimmykarily those other distributions were working differently, e.g. they don't want to ship the mesh network functionality at all, or doing something else I'm not aware of. In this case, it would be about having the same level of functionality at the provider-kairos level between the two distributions

Reply:
Ok, we've decided to remove the interactive installer until the package is sorted out, so there's no case when someone using the interactive installer selects k0s, but there's no k0s package https://github.com/kairos-io/kairos/issues/3124